build-logic/src/main/java/org/ehcache/build/conventions/DeployConvention.java

```diff
@@ -76,7 +76,7 @@ public void apply(Project project) {
         pom.developers(devs -> devs.developer(dev -> {
           dev.getName().set("Terracotta Engineers");
           dev.getEmail().set("tc-oss@softwareag.com");
```

Comment:
shouldn't tc-oss@softwareag.com be removed?

Reply:
Replaced with `tc-oss-dg@ibm.com`